js/service/registration.js

```diff
@@ -0,0 +1,63 @@
+import {nc_fetch_json} from 'nextcloud_fetch';
+
+export function getVerificationState () {
+	let url = OC.generateUrl('/apps/twofactor_email/settings/state')
```

Comment:
After this gets merged @ChristophWurst can confert this into his fancy tiny package stuff